src/clusterfuzz/_internal/bot/fuzzers/afl/launcher.py

```diff
@@ -932,7 +999,11 @@ def afl_output(self):
     reproducing testcases."""
 
     if self._afl_output is None:
-      self._afl_output = AflFuzzOutputDirectory()
+      if environment.is_android():
```

Comment:
Can you reverse this condition so that the more common case is not in "else"
i.e.
```
if not environment.is_android():
...
else:
...
```